package.json

```diff
@@ -51,14 +51,15 @@
   },
   "scripts": {
     "postinstall": "yarn --cwd ./examples",
-    "prebuild": "rimraf dist && npm run typegen",
-    "build": "rollup -c && yarn copy && rimraf dist/__tests__",
+    "build": "rimraf dist && npm run typegen && rollup -c && yarn copy && rimraf dist/__tests__ && npm run replace-reconciler-imports",
     "prepare": "npm run build",
     "eslint": "eslint --fix src/**/*.{ts,tsx}",
     "dev": "yarn --cwd ./examples start",
-    "test": "jest",
-    "typecheck": "tsc --noEmit --emitDeclarationOnly false --strict --jsx react",
-    "typegen": "tsc && mv dist/src/* dist && rm -rf dist/src || true",
+    "tests": "jest",
+    "test": "npm run typecheck && npm run eslint && npm run tests && npm run build",
+    "typecheck": "tsc --noEmit",
+    "replace-reconciler-imports": "npx replace 'import Reconciler' 'import * as Reconciler' dist -r --include='*.d.ts'",
+    "typegen": "tsc --declaration --emitDeclarationOnly --declarationDir dist && mv dist/src/* dist && rm -rf dist/src || true",
```

Comment:
IMHO using raw unix file commands in an npm script is not too good (Not regarding this specific pr)